B/boostpython/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "boostpython"
+version = v"1.76.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/boostorg/python.git", "68fa9dccdec75d59ac28d3ff98d70c659e12326d")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/python
+g++ -fPIC -o $libdir/libboost_python.$dlext -O3 -shared -L$libdir -lpython3.8 -I$includedir -I$includedir/python3.8 $(find src ! -path src/numpy/\* -name \*.cpp)
```

Comment:
Build with Clang on BSD systems?
```suggestion
c++ -fPIC -o $libdir/libboost_python.$dlext -O3 -shared -L$libdir -lpython3.8 -I$includedir -I$includedir/python3.8 $(find src ! -path src/numpy/\* -name \*.cpp)
```